examples/explicit.py

```diff
@@ -0,0 +1,31 @@
+import datetime
+import redis
+
+import lightstep
```

Comment:
Do we want to directly use LightStep in this tracer example? Can we avoid requiring any OT implementation and just show some kind of "set up your tracer" placeholder?